draft-kazuho-httpbis-priority.md

```diff
@@ -509,7 +509,13 @@ the server did not specify the `progressive` parameter.
 
 # Security Considerations
 
-## Fairness and Coalescing Intermediaries {#fairness}
+## Fairness {#fairness}
+
+Client initiated priority SHOULD be scoped to the client connection: Allowing
+Client_A requests to directly impact the scheduling of Client_B requests
+creates a fairness problem.
```

Comment:
I would rewrite this slightly to say something like

"for example, in a case where there are two clients A and B; allowing requests from Client A to directly impact the scheduling of responses to Client B's requests creates a fairness problem.

Reply:
Minor tweak on top.